ignite/metrics/gan/utils.py

```diff
@@ -23,7 +24,12 @@ def __init__(self, return_features: bool, device: Union[str, torch.device] = "cp
             raise RuntimeError("This module requires torchvision to be installed.")
         super(InceptionModel, self).__init__()
         self._device = device
-        self.model = models.inception_v3(pretrained=True).to(self._device)
+        if Version(torch.__version__) <= Version("1.7.0"):
+            model_kwargs = {"pretrained": True}
+            self.model = models.inception_v3(model_kwargs).to(self._device)
+        else:
+            self.model = models.inception_v3(weights=models.Inception_V3_Weights.DEFAULT).to(self._device)
```

Comment:
@puhuk this looks bad and the code wont work for True-branch of `if Version(torch.__version__) <= Version("1.7.0")`.
I previously suggested to have a single call:
```python
self.model = models.inception_v3(**model_kwargs).to(self._device)
```
Can you make this ?